repos/system_upgrade/el7toel8/actors/storagescanner/libraries/library.py

```diff
@@ -87,14 +87,23 @@ def _get_fstab_info(fstab_path):
                 if not entry:
                     continue
 
-                fs_spec, fs_file, fs_vfstype, fs_mntops, fs_freq, fs_passno = entry.split()
-                yield FstabEntry(
-                    fs_spec=fs_spec,
-                    fs_file=fs_file,
-                    fs_vfstype=fs_vfstype,
-                    fs_mntops=fs_mntops,
-                    fs_freq=fs_freq,
-                    fs_passno=fs_passno)
+                entries = entry.split()
+
+                if len(entries) == 4:
+                    entries.append('0')
+
+                if len(entries) == 5:
+                    entries.append('0')
+
+                if len(entries) == 6:
```

Comment:
The logic should be different. If you end up with anything else then 6 entries, you should:
- log error & raise stopactorexecutionerror
- or log error and  inhibit upgrade with a msg that fstab seems to be wrong and user should check it. Would be even nice to print the line (or anything that would point what line is problematic at all) so msg will not be confusing so much and user could fix it (or we will be able to investigate it better)

Reply:
I agree with @pirat89 . It would be nice to log error and inhibit upgrade in case there is a line in fstab with anything else then 6 entries.